### PR TITLE
Add acceptance test for EE Ticket #3241

### PR DESCRIPTION
**Pull request by micbar**
## Description
Fix for owncloud/enterprise#3241

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by PVince81** [comment]
@individual-it @phil-davis please review

**Comment by micbar** [comment]
@phil-davis @individual-it The core PR  https://github.com/owncloud/core/pull/35156/ is needed to make this work.

File: tests/acceptance/features/bootstrap/GuestsContext.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ private function setCSRFDotDisabledFromGuestsScenario($setting) {
 	 * @return string
 	 */
 	public function prepareUserNameAsFrontend($guestEmail) {
-		return \strtolower(\trim(\urldecode($guestEmail)));
```
Comment (individual-it) [inline tests/acceptance/features/bootstrap/GuestsContext.php]: why do we need that change?

Comment (micbar) [reply tests/acceptance/features/bootstrap/GuestsContext.php]: If we always convert the UserID to lowercase, we cannot test real world scenarios where the users enter Uppercase letters as part of the E-Mail Address.

**Comment by micbar** [comment]
Needs to be decided. There are issues with the lowercase usernames.

**Comment by micbar** [comment]
@phil-davis let us close this.
The case is not easy to test, because the fix is in the UI / JS which converts the e-mail to lowercase before making the request to the sharing api.

**Pull request closed without merging**